### DISCUSSION TITLE
v1.22.2

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#264] [Announcements]
(Hotfix for 1.22.0, fixes search result order and restores the old B/W color scheme; hotfix for 1.22.1, fixes duplicate search results and launch on enter not working in pager layout)

- Themed icons compat: The launcher now supports native themed app icons on all Android versions
- Added support for themed icon packs
- Added themed launcher icon (thanks to @Sir-Photch)
- The clock widget can now show the favorites component and one additional component at the same time (thanks to @Sir-Photch)
- Redesigned weather widget (thanks to @Sir-Photch)
- Redesigned music widget
- Added an option to launch the first search result with the keyboard's enter key (thanks to @Sir-Photch)
- Added an option to lock the launcher's orientation to portrait mode (thanks to @Sir-Photch)
- Updated Outfit font; no more missing glyphs in Polish, Czech, Romanian, Turkish, and several other languages
- Bugfixes

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.22.2'>v1.22.2</a>.</em>

**Comment by laurensdc** [comment]
Thank you for your continued work!! This launcher is absolutely amazing.

Are there any performance updates on the roadmap? Cheers!

**Reply by jspivey40** [reply]
> Thank you for your continued work!! This launcher is absolutely amazing.
> 
> Are there any performance updates on the roadmap? Cheers!

Same thing I'm hoping to see soon. The only thing that got me using another launcher is performance issues. Definitely is going to be pretty awesome with some perf gains

**Reply by shtrophic** [reply]
I'm afraid just "performance issues" is not specific enough for anyone to start diagnosing this issue. 

- What kind of issue, when does it happen, how can in be reproduced?
- _What device?_
